src/main/java/edu/texas/social_computing/hospitals/Resident.java

```diff
@@ -6,6 +6,15 @@
     private Resident partner;
     private List<Hospital> preferences;
     private List<Hospital> excludedHospitals;
+    private Hospital assignedHospital;
+
+    public Hospital getAssignedHospital() {
```

Comment:
I thought we were going to have the assignment defined by the hospital's list of residents, and create a `getAssignmentForResident(List<Hospital>)` method in the utils class. That way we only track/change it in one place, less error prone.

Reply:
ok, then let's make static util methods `assign(Resident, Hospital)` and `unassign(Resident, Hospital)` that take care of both the Resident and Hospital fields at the same time.

Reply:
Or manage the assignments externally, in a `Multimap` or something.